src/stream/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./share";
```

Comment:
Why did you call the directory "stream"?

Reply:
I was trying to find a name that would work for import-export-share… Couldn't find anything better

Reply:
I wasn't even able to guess it even though I know what the code does. How about `import-export`?